packages/docs/package.json

```diff
@@ -1,88 +1,89 @@
 {
-	"name": "@availity/docs",
```

Comment:
the reason for the huge diff on this file is a formatting change. the real change is I added a dev dependency, `@availity/step-wizard` @GoPro16